colin/cli/colin.py

```diff
@@ -80,7 +82,7 @@ def cli():
               help="Timeout for each check in seconds. (default=600)")
 @click.option('--insecure', is_flag=True, default=False,
               help="Pull from an insecure registry (HTTP or invalid TLS).")
-def check(target, parent_target, ruleset, ruleset_file, debug, json, stat, skip, tag, verbose,
+def check(target, parent_target, ruleset, ruleset_file, debug, json, xunit, stat, skip, tag, verbose,
```

Comment:
Could you add verbose to next line?

https://app.codacy.com/manual/user-cont/colin/pullRequest?prid=5390981